tools/Google.Cloud.Tools.ReleaseManager/Program.cs

```diff
@@ -31,57 +26,34 @@ namespace Google.Cloud.Tools.ReleaseManager
     /// </summary>
     internal sealed class Program
     {
-        // The branch we're thinking of as master. It may actually be some other branch for occasional
-        // releases, but fundamentally it's "the current source of truth we're basing this release on".
-        private const string MasterBranch = "master";
-
-        private const string ShowVersionCommand = "show-version";
-        private const string SetVersionCommand = "set-version";
-        private const string IncrementVersionCommand = "increment-version";
-        private const string ShowCommand = "show";
-        private const string CompareCommand = "compare";
-        private const string UpdateHistoryCommand = "update-history";
-        private const string CommitCommand = "commit";
-
         private static int Main(string[] args)
         {
             try
             {
+                var allCommands = typeof(Program).Assembly.GetTypes()
+                    .Where(t => !t.IsAbstract && typeof(ICommand).IsAssignableFrom(t))
```

Comment:
Not sure if you'll get the abstract BaseCommand here? Suppose not given that you are creating an instance on the Select and that would have failed for the abstract class.

Reply:
That's what the `!t.IsAbstract` is for :)